CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [2.3.1] - 2021-08-
```

Comment:
The version change should be a minor. You increase by a major and a minor and a bug fix simultaneously here.

Reply:
Changed to 1.3.2

Reply:
Please look into semantic versioning. https://semver.org/

The expected new version would be 1.3.0 since it is beeing increased to the next bigger minor version. 1.3.2 would suggest that there were two big fix releases before the current change.

Reply:
Changed to 1.3.0